backend/tfrs/settings.py

```diff
@@ -102,6 +103,7 @@
 ]
 
 WSGI_APPLICATION = 'wsgi.application'
+TEST_RUNNER = 'django_nose.NoseTestSuiteRunner'
```

Comment:
@rstens Can we confirm this will still run the unit tests we have?

Reply:
Yes, this will run all the tests.